Sum_of_numbers_string.py

```diff
@@ -0,0 +1,22 @@
+// ADDING ALL NUMBERS IN A STRING.
```

Comment:
Same issues as your other PR (comments, file location, reformat file using black, extra parentheses).

Reply:
Could you please let me know where to move the file to?

Reply:
a new folder in string algorithms.

Reply:
How do we reformat a file using black?